### PR TITLE
feat(TWILIGHT-2544): add pnpm lock file and update package manager to pnpm

### PR DESCRIPTION
**Pull request by Nabute**
What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)

* Feature: Added support for PNPM as the package manager and generated a lockfile.

What is the current behavior? (You can also link to an open issue here)

* Currently, the project uses npm, and the lockfile is generated as `package-lock.json`, which can lead to larger disk usage and slower installs due to duplication of dependencies across projects.

What is the new behavior? (You can also link to the ticket here)

* With this PR, the project switches to PNPM, which uses a global store for dependencies. The new `pnpm-lock.yaml` file ensures consistent dependency versions and faster installs.

Does this PR introduce a breaking change?

* No, this PR does not introduce any breaking changes. The project can continue to run with the same dependencies, just managed more efficiently with PNPM.

Screenshots (If appropriate)

* N/A

### CODE REVIEW

**Review by ashrafreda — APPROVED** [review]

**Comment by SallaDev** [comment]
> The branch `feature/TWILIGHT-2544-add-pnpm-support` of https://github.com/SallaApp/theme-raed/pull/468 has been synced from master..✅

File: public/product-card.js
Comment (ashrafreda) [inline public/product-card.js:1]: Remove this

File: public/product.js.LICENSE.txt
Comment (ashrafreda) [inline public/product.js.LICENSE.txt:1]: Remove this

File: public/testimonials.js.LICENSE.txt
Comment (ashrafreda) [inline public/testimonials.js.LICENSE.txt:1]: Remove this

File: public/product.js.LICENSE.txt
Comment (Nabute) [reply public/product.js.LICENSE.txt:1]: This commit removes the mentioned items as they no longer exist.

File: public/testimonials.js.LICENSE.txt
Comment (Nabute) [reply public/testimonials.js.LICENSE.txt:1]: This commit removes the mentioned items as they no longer exist.

**Pull request closed without merging**